src/pages/Auth/Auth.scss

```diff
@@ -0,0 +1,208 @@
+
+                input[type="text"],
+                input[type="email"],
+                input[type="password"] {
+                    display: block;
+                    width: 100%;
+                    border-radius: 0.25rem;
+                    border: 1px solid var(--text-color-light);
+                    padding: var(--sp-0-75);
+                    margin-bottom: var(--sp-0-5);
+                    font-size: var(--small-font-size);
+                }
+
+                .signup__container-cta {
+                    display: flex;
+                    gap: 1rem;
+                    align-items: center;
+
+                    label,
+                    input {
+                        margin: 0;
+                        cursor: pointer;
+                    }
+                }
+
+                .input__container-cta {
+                    display: flex;
+                    gap: 0.5rem;
+                    align-items: center;
+                    justify-content: space-between;
+
+                    .checkbox {
+                        display: flex;
+                        gap: 1rem;
+                        align-items: center;
+                    }
+                    label,
+                    input {
+                        margin: 0;
+                        cursor: pointer;
+                    }
+                }
+
+                button {
+                    color: var(--white-color);
+                    background-color: var(--title-color);
+                    width: 100%;
+                    padding: 1rem;
+                    margin: 1rem 0;
+                    border-radius: 0.25rem;
+                    cursor: pointer;
+
+                    &:hover {
+                        background-color: var(--dark-color);
+                    }
+                }
+
+                .login__container {
+                    text-align: center;
+                }
+
+                .legal {
+                    margin: 0.5rem 0;
+                    .small {
+                        font-size: var(--smaller-font-size);
+                    }
+                }
+
+                a {
+                    color: var(--secondary-color);
+                }
+            }
+
+            label {
+                display: block;
+                margin-bottom: var(--sp-0-25);
+                font-size: var(--smaller-font-size);
+            }
+        }
+    }
+}
+
+@media screen and (min-width: 768px) {
+    .auth_container {
+        display: grid;
+        grid-template-columns: 1fr;
+        margin-top: 4rem;
+        // background-color: red;
+        .split-screen {
+            flex-direction: row;
+            height: calc(100vh - 4rem);
+            // margin-top: var(--header-height);
+            // margin-top: 3rem;
+            // max-height: calc(100vh - var(--header-height));
+            // height: calc(100vh - var(--header-height));
+            // height: calc(100vh - var(--header-height) + 0.7rem);
```

Comment:
you can remove this if not needed

Reply:
Thanks, I missed this part while removing commented code